README.md

```diff
@@ -1 +1,13 @@
 # Laravel-Merged
+### installation
+- `composer install`
+- `npm install`
+- `npm run dev`
+- `cp .env.example .env`
+- `php artisan key:generate`
+- insert your db / other  credentials into `.env` file
+- `php artisan migrate`
+- `php artisan serve`
+
```

Comment:
also you need to mention it needs `hub` command as a pre-requests. and run `php artisan laravel:update` then `php artisan pr:fetch`

Reply:
more details about hub please? what should I run `php artisan hub` ?

Reply:
I got it you mean this https://snapcraft.io/hub

Reply:
I got this error `   Exception  : Repository not found: /home/moataz/apps/laravel-merge/framework
`

Reply:
may I need to configure `hub` first I tried to clone a repo I got this 
```
app: hub clone github/hub 
github.com username: ^C
 app: 

```

Reply:
I completed and it seems to work

Reply:
`php artisan laravel:update` works well 

but `php artisan pr:fetch`

 laravel-merge: php artisan pr:fetch
Loading merged PRs
parsing
Could not parse this line: git: 'pr' is not a git command. See 'git --help'.
Could not parse this line: Did you mean one of these?
Could not parse this line: pf
Could not parse this line: pl
filtering
inserting new PRs

Reply:
@ahmedash95 
```
git --version
git version 2.7.4
hub version 2.2.9

```

Reply:
Hey @MoatazAbdAlmageed, I've updated the readme of the repo. please check it and then update your PR with any conflict.

Thanks buddy.